global_variable_generator_old.py

```diff
@@ -0,0 +1,459 @@
+"""
+    Global Variable Generator:
+
+    Function:
+    - generate a csv file that contains all shelf-related global variable
+      based on the inputs in global_variable_template.xlsx
+    - csv file can be imported into Delta ISPSoft
+
+    How to use:
+    - configure parameter in global_variable_template.xlsx
+    - run this script at the same directory as global_variable_template.xlsx
+    - global_variable_table.csv will be generated upon script completion
+"""
+
+from typing import Union
+import os
+import csv
+import pandas as pd
+import numpy as np
+
+
+def main():
+
+    # parameter
+    input_name = "global_variable_template_old.xlsx"
+    global_var_table_name = "global_variable_table.csv"
+    hmi_tag_table_name = "hmi_tag.csv"
+    hmi_tag_plc_name = "{EtherLink1}1@"
+

```

Comment:
This global_variable_generator_old.py is not taking argument for filename?
How does it run the test then?

Reply:
input_name = "global_variable_template_old.xlsx"

It's taking in the old template. It's not taking in argument as it will always take in the same template.